storage/rocksdb/rdb_compact_filter.h

```diff
@@ -76,6 +76,7 @@ class Rdb_compact_filter : public rocksdb::CompactionFilter {
     } else if (m_ttl_duration > 0 &&
                should_filter_ttl_rec(key, existing_value)) {
       m_num_expired++;
+      global_stats->rows[ROWS_EXPIRED].inc();
```

Comment:
@hermanlee  can you elaborate what you mean by 

> add an api to ha_rocksdb_proto.h that lets you update a specific counter?

By exposing global_stats you can already update whatever operation type.  
I have moved the stats upload to the destructor.